client/landing/gutenboarding/components/domain-picker/style.scss

```diff
@@ -137,15 +137,12 @@
 			}
 		}
 	}
-}
-
-.domain-picker__suggestion-item-name {
-	letter-spacing: 0.4px;
 
 	input[type='radio'].domain-picker__suggestion-radio-button {
 		width: 16px;
 		height: 16px;
 		margin: 0 12px 0 0;
+		min-width: 16px; // prevents long domain from squishing the radio button
```

Comment:
Would using flex-shrink/grow help here without using fixed pixel value?

Reply:
Doesn't work as the input is at the mercy of flexbox's calculation.
Alternative solution is to nest this `<input>` tag inside a `<div>` parent, so the `<div>` plays the role of a flex item instead of this input element.
Between nesting this in a div or using `min-width`, I choose the latter. :)